financial-templates-lib/price-feed/CryptoWatchPriceFeed.js

```diff
@@ -13,10 +13,11 @@ class CryptoWatchPriceFeed extends PriceFeedInterface {
   // getTime function to return the current time.
   // minTimeBetweenUpdates min number of seconds between updates. If update() is called again before this number of
   // seconds has passed, it will be a no-op.
-  constructor(web3, logger, apiKey, exchange, pair, lookback, networker, getTime, minTimeBetweenUpdates) {
```

Comment:
Unrelated change but updated the price feed to make it more consistent with the rest of our infra. The logger should come as the first param.